src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -10,6 +10,7 @@
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.Text;
 using Roslyn.Utilities;
+using System.Threading.Tasks;
```

Comment:
Yup, missed a few things from cleanup when I was debugging. Should be fixed now.